django_mongodb/query.py

```diff
@@ -40,8 +40,13 @@ def __init__(self, compiler, columns):
         self.columns = columns
         self._negated = False
         self.ordering = []
+        self.collection_name = self.compiler.collection_name
         self.collection = self.compiler.get_collection()
         self.mongo_query = getattr(compiler.query, "raw_query", {})
+        # maybe I have to create a new object or named tuple.
+        # it will save lookups, some filters (in case of inner) and project to rename field
+        # don't know if the rename is needed
+        self.mongo_lookups = None
```

Comment:
I'm not sure if `mongo_lookups` is the best name. It kind of confuses me with Django lookups / `lookups.py`. Maybe something like `lookup_pipeline`?